cmd/juju/cloud/addcredential.go

```diff
@@ -62,8 +65,26 @@ Examples:
     juju add-credential google
     juju add-credential aws -f ~/credentials.yaml
 
+Cloud-specific instructions:
```

Comment:
The google instructions seem very specific and could easily become outdated if google changes the workflow on their end. Could we make the doc more generic. We just need to tell the user to obtain a json file or oauth2 token using the GCE web console.

Reply:
We can't really leave it as "obtain a json file or oauth2 token using the GCE web console". There are several methods for obtaining both. As indicated from the bug report, one of Juju's most experienced users wasn't able to figure that out.

I would like to be more helpful, even if there is a risk that things become stale. It's very easy to update documentation. It's very frustrating being a new user to Juju and getting stuck with credential issues.

Adding credentials should be as seemless as possible. At this stage, a new user is least likely to report any issues. They're nervous and undecided about whether Juju is a good tool.

My thinking after @pmatulis's comment was to remove the "how to" and keep things as "what is required". Juju's `jsonfile` means service account key in Google's terminology. Removing the steps to get one, but leaving in terms that are searchable might work.

Reply:
I've actually remove that addition to the help docs. I've changed the message in the interactive prompt so that it's clearer what's being asked for.